#######################################################################
# This message was automatically generated by the lcm_to_ros package
# https://github.com/nrjl/lcm_to_ros, nicholas.lawrance@oregonstate.edu
#######################################################################
#
# Source message:    .msg
# Creation:          Thu 16 Jul 2020 12:36:37 PM EDT
#
#######################################################################
string              str
bool                flag
byte                gnash
int8                i8_scalar
int8[2]             i8_array
int16               i16_scalar
int16[2]            i16_array
int32               i32_scalar
int32[2]            i32_array
int64               i64_scalar
int64[2]            i64_array
float32             flt_scalar
float32[2]          flt_array
float64             dbl_scalar
float64[2]          dbl_array
float64[]           variable_length_array
int32               num_channels
simple_channel[]    channels
